x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -276,5 +315,342 @@ export function getDetailsItems(anomaly: AnomaliesTableRecord, filter: EntityCel
+    defaultMessage: 'No',
+  });
+
+  const explanationDetails = [];
+  const anomalyType = getAnomalyType(explanation);
+  if (anomalyType !== null) {
+    explanationDetails.push({
+      title: (
+        <FormattedMessage
+          id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyType"
+          defaultMessage="Anomaly type"
+        />
+      ),
+      description: <>{anomalyType}</>,
+    });
+  }
+
+  if (scoreDifference > ACCEPTABLE_NORMALIZATION) {
+    explanationDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.recordScoreTooltip',
+            {
+              defaultMessage: 'Record score normalized blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.recordScore"
+              defaultMessage="Record score normalized"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: (
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <RecordScore score={initialScore} />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>{` -> `}</EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <RecordScore score={finalScore} />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+    });
+  }
+
+  const impactDetails = [];
+  if (explanation.anomaly_characteristics_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristicsTooltip',
+            {
+              defaultMessage: 'Anomaly characteristics impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristics"
+              defaultMessage="Anomaly characteristics impact"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: <ImpactVisual score={explanation.anomaly_characteristics_impact} />,
+    });
+  }
+
+  if (explanation.single_bucket_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.singleBucketTooltip',
+            {
+              defaultMessage: 'Single bucket impact blah blah blah',
```

Comment:
Description text needed

Reply:
@szabosteve and @valeriy42 can you please help me with this tooltip text.

Reply:
Updated in https://github.com/elastic/kibana/pull/142999/commits/45313cddd3db35b4d6d83f8b238aa9367e9953fa